app/models/provider_authorization.rb

```diff
@@ -157,10 +157,6 @@ def refresh_access_token!
        'Content-Type' => 'application/x-www-form-urlencoded'
      }
     }
-    if CONFIG.ca_path || CONFIG.ca_file
-      options[:ssl_ca_file] = CONFIG.ca_file
-      options[:ssl_ca_path] = CONFIG.ca_path
-    end
```

Comment:
Yep, noted. We weren't using this in production or Staging, and I'm not using it locally, so I was just trying to trim the fat. Easy to add back if we want